app/src/main/java/com/example/scoutconcordia/DataStructures/Graph.java

```diff
@@ -8,24 +8,29 @@
 import java.util.InputMismatchException;
 import java.util.Scanner;
 
+import static android.location.Location.distanceBetween;
+
+
 // Assumption is no two points are equivalent
 public class Graph
 {
     private int numberOfNodes;
     private Node[] nodes;
     private N_aryTree breathFirstSearchResults;
 
-    private class Node
+    public class Node
```

Comment:
Why was the inner Node class changed from private to public?

Can you explain any differences and your reasoning for this decision?

Reply:
I needed to loop through Graph nodes on line 254 in MapsActivity. If Nodes was private I was not able to do this.